src/config1.js

```diff
@@ -0,0 +1,15 @@
+// const url = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/';
+// const postData = async() => {
+//   const body = JSON.stringify({});
+//   const response = await fetch(url, {
+//     method: 'POST',
+//     body,
+//     headers: {
+//       'Content-type': 'application/json; charset=UTF-8',
+//     },
+//   });
+//   const data = await response.json();
+//   return data.result;
+// };
+
```

Comment:
Kindly remove this file if you are not using it. Commented code makes your project looks messy. :+1: :smiley: :smiley: